src/libs/actions/Link.ts

```diff
@@ -10,29 +8,23 @@ import * as Url from '../Url';
 let isNetworkOffline = false;
 Onyx.connect({
     key: ONYXKEYS.NETWORK,
-    callback: (val) => (isNetworkOffline = lodashGet(val, 'isOffline', false)),
+    callback: (value) => (isNetworkOffline = value?.isOffline ?? false),
 });
 
-let currentUserEmail;
+let currentUserEmail: string;
```

Comment:
```suggestion
let currentUserEmail: string | undefined;
```

I would be more safe if we specify that this variable can be undefined, because it's only populated in the callback of Onyx, so before that happens it's undefined and we should consider this possibility.

Reply:
Or if we only care about the `string` type, maybe we can default it to `''`? Wdyt?

Reply:
It is an option too, @kubabutkiewicz We can change this one to:

```ts
let currentUserEmail = '';
```